Pi3BoardPkg/Pi3BoardPkg.dsc

```diff
@@ -643,8 +643,10 @@
   #
   # MMC/SD
   #
-  Pi2BoardPkg/Drivers/MmcDxe/MmcDxe.inf
-  Pi2BoardPkg/Drivers/SdHostDxe/SdHostDxe.inf
+  #Pi3BoardPkg/Drivers/MmcDxe/MmcDxe.inf
```

Comment:
Do we expect anyone to uncomment these lines in the future? If not, please remove.